source/frontend/src/components/maps/leaflet/Layers/useConfiguredMapLayers.tsx

```diff
@@ -19,9 +20,9 @@ export const useConfiguredMapLayers = () => {
 
     return {
       ...parent,
-      nodes: allNodes?.map((node: any, index) => ({
+      nodes: allNodes?.map((node: ILayerItem, index) => ({
         ...node,
-        zIndex: (parentIndex + 1) * index,
+        zIndex: node.zIndexAbsolute === true ? node.zIndex : (parentIndex + 1) * index,
```

Comment:
I was unsure about having this as just node.zIndex, but I presume the index calculation was added to solve some previous issue. If that does not ring a bell I can simplify it and remove the zIndexAbsolute property

Reply:
I'm pretty sure this logic predates PSP (ie. is from CITZ pims). I suppose this makes sense if the layers are ordered into groups by priority, which isn't really our case. However, we also don't have a defined priority for our existing layers - which sounds like a requirements gap. 

Logged: https://jira.th.gov.bc.ca/browse/PSP-9212 for the above.